### ISSUE TITLE
Uploads don't work with Room v6

### ISSUE THREAD

**Issue by commagray** [vector-im/riotX-android#1558]
Hi!

I can't send attachments via RiotX v0.22.0 in rooms v6.

![image](https://user-images.githubusercontent.com/27947961/85927896-a4b0e500-b8b1-11ea-92fa-d4b2241c83bd.png)

Attachment above was uploaded via legacy Riot.im app. Attachment below — via RiotX. Also, uploads work with FluffyChat, nheko, Mirage… so pretty much everything else except just RiotX. Uploads work in rooms version 5 or below.

My server is https://gospel.sunbutt.faith. It uses Synapse v1.15.1 with matrix-media-repo v1.1.2 for managing attachments.

I'm not the only affected user. Folks in #degeneral:sunbutt.faith complain that attachments don't work after upgrading from v5 to v6.

Thanks.

UPD. Just updated beta of Riot.im (which is RiotX underhood) is affected too.
UPD2. Tested with matrix.org account and the thing stays the same.

**Comment by anoadragon453** [comment]
I can confirm this is also happening to me in a test v6 room.

Logcat:

```
06-28 14:43:16.500 28696 29198 V QueryStringValueProcessorKt: No condition to process
06-28 14:43:16.519 28696 28696 I VectorBaseBottomSheetDialogFragment: onResume BottomSheet MessageActionsBottomSheet
06-28 14:43:16.567 28696 28825 V QueryStringValueProcessorKt: No condition to process
06-28 14:43:16.600 28696 28842 W ExifInterface: Invalid image: ExifInterface got an unsupported image format file(ExifInterface supports JPEG and some RAW image formats only) or a corrupted JPEG file to ExifInterface.
06-28 14:43:16.600 28696 28842 W ExifInterface: java.io.IOException: Invalid byte order: ffff8950
06-28 14:43:16.600 28696 28842 W ExifInterface:         at android.media.ExifInterface.readByteOrder(ExifInterface.java:3128)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at android.media.ExifInterface.isOrfFormat(ExifInterface.java:2443)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at android.media.ExifInterface.getMimeType(ExifInterface.java:2321)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at android.media.ExifInterface.loadAttributes(ExifInterface.java:1755)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at android.media.ExifInterface.<init>(ExifInterface.java:1449)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.resource.bitmap.ExifInterfaceImageHeaderParser.getOrientation(ExifInterfaceImageHeaderParser.java:40)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.ImageHeaderParserUtils.getOrientation(ImageHeaderParserUtils.java:92)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.resource.bitmap.Downsampler.decodeFromWrappedStreams(Downsampler.java:269)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.resource.bitmap.Downsampler.decode(Downsampler.java:224)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.resource.bitmap.Downsampler.decode(Downsampler.java:173)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.resource.bitmap.ByteBufferBitmapDecoder.decode(ByteBufferBitmapDecoder.java:31)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.resource.bitmap.ByteBufferBitmapDecoder.decode(ByteBufferBitmapDecoder.java:14)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodePath.decodeResourceWithList(DecodePath.java:92)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodePath.decodeResource(DecodePath.java:70)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodePath.decode(DecodePath.java:59)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.LoadPath.loadWithExceptionList(LoadPath.java:76)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.LoadPath.load(LoadPath.java:57)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodeJob.runLoadPath(DecodeJob.java:524)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodeJob.decodeFromFetcher(DecodeJob.java:488)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodeJob.decodeFromData(DecodeJob.java:474)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodeJob.decodeFromRetrievedData(DecodeJob.java:426)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodeJob.onDataFetcherReady(DecodeJob.java:390)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DataCacheGenerator.onDataReady(DataCacheGenerator.java:94)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.model.ByteBufferFileLoader$ByteBufferFetcher.loadData(ByteBufferFileLoader.java:70)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DataCacheGenerator.startNext(DataCacheGenerator.java:74)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodeJob.runGenerators(DecodeJob.java:310)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodeJob.runWrapped(DecodeJob.java:276)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.DecodeJob.run(DecodeJob.java:234)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at java.lang.Thread.run(Thread.java:919)
06-28 14:43:16.600 28696 28842 W ExifInterface:         at com.bumptech.glide.load.engine.executor.GlideExecutor$DefaultThreadFactory$1.run(GlideExecutor.java:431)
06-28 14:43:18.242 28696 28815 D OpenGLRenderer: endAllActiveAnimators on 0x79440d0400 (ConstraintLayout) with handle 0x798a631b40
06-28 14:43:18.249   895  7770 E InputDispatcher: Window handle Window{85f70e5 u0 im.vector.riotx.debug/im.vector.riotx.features.home.room.detail.RoomDetailActivity} has no registered input channel
06-28 14:43:18.265   895  9621 E InputDispatcher: Window handle Window{85f70e5 u0 im.vector.riotx.debug/im.vector.riotx.features.home.room.detail.RoomDetailActivity} has no registered input channel
06-28 14:43:18.308 28696 29199 V AsyncTransactionKt$awaitTransaction: Execute transaction in 46 millis
06-28 14:43:18.636 28696 29176 V TaskExecutor$execute$1$invokeSuspend$$inlined$runCatching$lambda: Enqueue task im.vector.matrix.android.internal.crypto.tasks.DefaultRoomVerificationUpdateTask with ID: e2ecd907-bf98-4199-8c6f-e9366ebf2efa
06-28 14:43:18.636 28696 28779 V EventRelationsAggregationUpdater: EventRelationsAggregationUpdater called with 1 insertions
06-28 14:43:18.636 28696 29176 V TaskExecutor$execute$1$invokeSuspend$$inlined$runCatching$lambda: Execute task im.vector.matrix.android.internal.crypto.tasks.DefaultRoomVerificationUpdateTask with ID: e2ecd907-bf98-4199-8c6f-e9366ebf2efa on DefaultDispatcher-worker-3
```

**Comment by Bubu** [comment]
Duplicate of #363 ?

**Comment by bmarty** [comment]
Should be fixed by #1766
